packages/web/hooks/use-feature-flags.ts

```diff
@@ -28,6 +28,29 @@ type ModifiedFlags =
   | "_isInitialized"
   | "_isClientIDPresent";
 
+// These default flags will be passed to the app if the app is running in development mode
```

Comment:
Would also be nice to add a note that these defaults are for external contributors. Team members should prefer the `NEXT_PUBLIC_LAUNCH_DARKLY_CLIENT_SIDE_ID` with the dev ID set so that they can test with LD itself.